projects/valtimo/form-management/src/lib/components/form-management-list/form-management-list.component.ts

```diff
@@ -25,7 +25,7 @@ import {
   CarbonListModule,
   ColumnConfig,
   ConfirmationModalModule,
-  Pagination,
+  Pagination, ViewType,
```

Comment:
Please format. Probably you should configure Intellij to format automatically with prettier on save